team-fingerprints-web/src/components/EmailForm/styles.sass

```
@@ -38,6 +38,16 @@
     &__container
       max-height: 300px
       overflow: scroll
+      overflow-x: hidden
+
+      &::-webkit-scrollbar 
+        width: 12px
+      
+
+      &::-webkit-scrollbar-thumb 
+        background-color: #2a282b
+        border-radius: 10px
```

Please use sass variables instead
How about using some already defined `background-color` to stay consistent throughout the app like `$color-bglight` or `$color-white26`
```suggestion
      &::-webkit-scrollbar-thumb 
        background-color: #2a282b
        border-radius: $border-radius-sm
```

---

You're correct, I've changed to scss variables and added declared color